src/components/SetKeystorePasswordDialog.vue

```diff
@@ -115,41 +116,37 @@ export default {
   computed: {
     _show: {
       get() {
-        return this.show;
+        return this.show
       },
       set(val) {
-        this.$emit("toggle", val);
+        this.$emit("toggle", val)
       }
     },
     sitekey() {
-      return process.env.VUE_APP_RECAPTCHA_SITE_KEY;
+      return process.env.VUE_APP_RECAPTCHA_SITE_KEY
     },
     ...mapState({
       substrateApi: (state) => state.substrate.api,
       isLoading: (state) => state.substrate.isLoadingWallet
     }),
     nameRules() {
-      return [
-        val => !!val || "This field is required",
-        val => (val && val.length >= 8) || "Min 8 Character"
-      ]
+      return [rulesHandler.FIELD_REQUIRED, rulesHandler.MAX_CHARACTER(100)]
     },
     passwordRules() {
       return [
-        val => !!val || "This field is required",
-        val => (val && val.length >= 8) || "Password Min 8 Character",
-        val => /^[a-zA-Z0-9-_]+$/.test(val) || "Password must a-z, A-Z, "
+        rulesHandler.FIELD_REQUIRED,
+        rulesHandler.PASSWORD
       ]
     }
   },
   mounted() {
-    let recaptchaScript = document.createElement("script");
+    let recaptchaScript = document.createElement("script")
     const recaptchaSrc =
-      "https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit";
-    recaptchaScript.setAttribute("src", recaptchaSrc);
-    recaptchaScript.setAttribute("async", true);
-    recaptchaScript.setAttribute("defer", true);
-    document.head.appendChild(recaptchaScript);
+      "https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit"
+    recaptchaScript.setAttribute("src", recaptchaSrc)
+    recaptchaScript.setAttribute("async", true)
+    recaptchaScript.setAttribute("defer", true)
+    document.head.appendChild(recaptchaScript)
```

Comment:
Aren't we using `vue-recapthca`? Why does there need to be a manual script here?

We didn't need this on our `ui-components` library: https://github.com/debionetwork/ui-components/blob/main/src/components/UiDebioRecaptcha.vue